app/assets/stylesheets/spotlight/_blacklight_overrides.scss

```diff
@@ -8,13 +8,9 @@
   .input-group {
     width: 100%;
   }
-
-  .input-group-addon {
-    &.for-search-field {
-      background-color: $input-bg;
-      border-radius: $border-radius 0 0 $border-radius;
-      width: 15ch;
-    }
+  
+  .search-btn {
+    max-height: 2.375rem;
```

Comment:
Just curious, where does 2.375 come from?

Reply:
Its the calculated rem value for 38px when using a base of 16px. We could use 38px here instead but if someone changes the default font size i think it would be wrong.